### PR TITLE
Fix touch query (missing $) causing Postgresql syntax error

### PR DESCRIPTION
**Pull request by enginefeeder101**
Hi,

First of all, thanks for this module. After I started using it my Node console and Postgresql logs started filling with syntax errors: `syntax error at or near "{"`. Enabling query logging identified the culprit.
A further look identified the source of the problem in the `touch`-method. This pull request should fix that.
The string literal references to `this.columns.session_id` were missing a `$`.

Regards

### CODE REVIEW

**Review by Samathingamajig — APPROVED** [review]

**Comment by Fezzzi** [comment]
@enginefeeder101 thanks a lot for finding and fixing this issue 🥇 , could either you or @Samathingamajig please merge it? Would be very helpful!

**Comment by Samathingamajig** [comment]
@Fezzzi We need someone with write access to this repo to be able to merge it, and it appears that only @sanjaypojo has write access. If you need to use this fix temporarily while we're waiting for the merge & update to npm, you remove `express-pg-session` from your `package.json` and run `npm install git+https://github.com/enginefeeder101/express-pg-session.git#hotfix-touch` (or `yarn add ...` if you use yarn)

**Comment by sanjaypojo** [comment]
Thanks for the PR, I'm sorry I didn't see it earlier! Didn't get a notif since no one @mentioned me. Just merged it, will make a release when I get some time this week!

**Comment by enginefeeder101** [comment]
Thanks @sanjaypojo for merging! Will switch back from my fork to this repo in production!

**Pull request merged**
